pkg/apis/nodeup/config.go

```diff
@@ -76,6 +76,8 @@ type Config struct {
 
 	// APIServerConfig is additional configuration for nodes running an APIServer.
 	APIServerConfig *APIServerConfig `json:",omitempty"`
+	// Nvidia contains the configuration for nvidia
+	Nvidia *kops.NvidiaConfiguration
```

Comment:
Should it be committed if empty?
```suggestion
	Nvidia *kops.NvidiaConfiguration `json:",omitempty"`
```